packages/conversation/src/models/v1/lookup-capability-response/lookup-capability-response.ts

```diff
@@ -3,7 +3,7 @@ import { Recipient } from '../recipient';
 /**
  * An CapabilityResponse contains the identity of the recipient for which will be perform a capability lookup.
  */
-export interface QueryCapabilityResponse {
+export interface LookupCapabilityResponse {
```

Comment:
_My own interest here_  😇 
We could imagine having a mapping table to share same naming across SDKs (oas generator tool could be a good location)